spec/components/topic_view_spec.rb

```diff
@@ -694,4 +694,23 @@ def topic_view_near(post, show_deleted = false)
       expect(topic_view.last_post_id).to eq(p3.id)
     end
   end
+
+  describe '#read_time' do
+    let!(:post) { Fabricate(:post, topic: topic) }
+
+    before do
+      post.revise(Discourse.system_user, raw: "![image|100x100](upload://upload.png)")
```

Comment:
Why do you need to revise it, can't you pass the raw directly?

Reply:
It's a hack to make it update the word_count of the topic.